src/Artsy/Router/buildAppRoutes.tsx

```diff
@@ -34,11 +34,12 @@ export function makeAppRoutes(routeList: RouteList[]): RouteConfig[] {
        * manifest, navigate via router versus doing a hard jump between pages.
        */
       catchLinks(window, href => {
-        const url = ROUTE_NAMESPACE + href
```

Comment:
What makes `ROUTE_NAMESPACE` no longer required?

Reply:
It was always just an empty string -- I had this in place because for the POC I had apps mounted within a namespace, but no longer needed.